modules/@apostrophecms/schema/index.js

```diff
@@ -585,20 +592,22 @@ module.exports = {
         {
           fetchRelationships = true,
           ancestors = [],
-          isParentVisible = true
+          rootConvert = true,
+          ancestorSchemas = {},
+          ancestorPath = ''
         } = {}
       ) {
         const options = {
           fetchRelationships,
           ancestors,
-          isParentVisible
+          ancestorSchemas,
+          ancestorPath
```

Comment:
I think it is better to compute the `aposPath` of a field once, in the `validate` method of `schema` and its related functions, rather than pushing this boulder up the hill on each convert call 🪨

Reply:
Yes good point. We agree that we are talking about the schema path and not the actual destination one.

Reply:
Done, it's now computed during `validate`.